### ISSUE TITLE
Consider blocking modifications of contact/calendar entries if subscription has expired

### ISSUE THREAD

**Issue by Xiretza** [etesync/android#210]
If a user installed etesync using the trial license, then never renewed that license but still kept etesync installed, any modifications made to contact/calendar entries after expiration of the license will be irrecoverably lost when the app is uninstalled. It might make sense to block changes to the local cache when the license has expired, at least until the user explicitly acknowledges this risk, to avoid such mishaps.

**Comment by tasn** [comment]
I don't think we can actually block them (maybe we can make it read only, but that's probably a pain), though we pop up notifications alerting people of this fact. Do we not?

**Comment by Xiretza** [comment]
Yes, but apparently those notifications don't convey the possibility of data loss clearly enough, since a user has just run into this.

**Comment by tasn** [comment]
😱

**Comment by tasn** [comment]
I'm not sure how easy it is (or even whether possible) to change it to/from read only, so hard for me to comment on solutions.
